docs/_includes/common/tooltip.html

```diff
@@ -0,0 +1,147 @@
+    <p>Toggle the <span class="highlight">tourtip--expanded</span> modifier to expand or collapse the tourtip.</p>
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="tourtip tourtip--expanded">
+                <p class="tourtip__host">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                <div class="tourtip__overlay" style="right: 0; top: 25%" role="region" aria-labelledby="tourtip-label">
+                    <span class="tourtip__pointer tourtip__pointer--left"></span>
+                    <div class="tourtip__mask">
+                        <div class="tourtip__cell">
+                            <span class="tourtip__content">
+                                <h2 class="tourtip__heading" id="tourtip-label">Tourtip</h2>
+                                <p>Here's something new to help you be successful at your task.</p>
+                            </span>
+                            <button aria-label="Dismiss tourtip" class="tourtip__close" type="button">
+                                <span aria-hidden="true" role="img"></span>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="tourtip tourtip--expanded">
+    <p class="tourtip__host">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <div class="tourtip__overlay" role="region" aria-labelledby="tourtip-label">
+        <span class="tourtip__pointer tourtip__pointer--left"></span>
+        <div class="tourtip__mask">
+            <div class="tourtip__cell">
+                <span class="tourtip__content">
+                    <h2 class="tourtip__heading" id="tourtip-label">Tourtip</h2>
+                    <p>Here's something new to help you be successful at your task.</p>
+                </span>
+                <button aria-label="Dismiss tourtip" class="tourtip__close" type="button">
+                    <span aria-hidden="true" role="img"></span>
+                </button>
+            </div>
+        </div>
+    </div>
+</div>
+{% endhighlight %}
+
+</div>
```

Comment:
I still think we need something about all the pointer position options. Just a simple bullet list would suffice.

Reply:
Yeah. I'll put it under tooltip, rather than flyout notice, seeing as though flyout notice is deprecated.

Reply:
Correct. 👍